nonnull-interned-demo/daikon/testdemo/varinfo.0.expected

```diff
@@ -13,10 +21,10 @@ src/daikon/VarInfo.java:3805: error: [assignment.type.incompatible] incompatible
                                 ^
   found   : String
   required: @Interned String
-3 errors
+5 errors
 
 BUILD FAILED
 build.xml:9: The following error occurred while executing this line:
 ant.xml:16: Java returned: 1
 
-Total time: 6 seconds
+Total time: 13 seconds
```

Comment:
You shouldn't have to make this change; this line should be ignored by the diff.

Reply:
@mernst I have suppressed these warnings and so there is no change in the expected output file.